### PR TITLE
Feature/controller testing

### PR DESCRIPTION
**Pull request by simplegoose**
In this PR, the following was done:

- A spec file for testing the user controller was created.
- A spec file for testing posts controller was created.

### CODE REVIEW

File: spec/requests/users_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  let(:user) do
+    User.create(name: 'Harry', photo: 'https://photos.com/harrystyles', bio: 'harry Styles here!')
+  end
+
+  describe 'Get /' do
+    it 'Sends a success response' do
+      get '/'
+
+      expect(response).to have_http_status(:success)
+    end
```
Comment (thecodechaser) [inline spec/requests/users_request_spec.rb:13]: - [ ] Below tests are failing, Please make sure that all tests pass without any errors.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @simplegoose👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with controller specs. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] The issues page link is broken in the README file, Please make sure to fix this. It will make the README file more professional.

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_request_spec.rb (hunk 1 above)
Comment (simplegoose) [reply spec/requests/users_request_spec.rb:13]: The tests are passing from my end, maybe you haven't migrated the database. @thecodechaser 
Screenshot (62)

**Review by IngDavidTM — APPROVED** [review]
Hi @simplegoose 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**